modules/pipeline-manifest/bin/get-latest-manifest.sh

```diff
@@ -4,18 +4,20 @@
 #   $1 - name of the branch to get snapshots from (typically x.y-edge or x.y-stable)
 #   $2 - the Z release desired (i.e. 1.0.1, 2.0.1)
 #   $3 - the name of the pipeline repo to use (typically pipeline or backplane-pipeline)
+#   $4 - the name of the github organization
 #
 # Required environment variable:
 #   $GITHUB_TOKEN - the token.  To github.
 
 PIPELINE_MANIFEST_LATEST_BRANCH=$1
 PIPELINE_MANIFEST_LATEST_Z_RELEASE=$2
 PIPELINE_MANIFEST_REPO=$3
+PIPELINE_MANIFEST_ORG=$4
```

Comment:
This will be something that we need to be cognizant of in external automation references since the format will have changed

Reply:
Also we should probably be calling these via their Makefile targets, as that will populate the correct parameters - that's essentially their interface.